pom.xml

```diff
@@ -87,6 +92,18 @@
     <dependencyManagement>
         <dependencies>
             <!-- overrides of imports -->
+            <!-- FIXME: to be removed at next gridsuite-dependencies upgrade  -->
+            <dependency>
+                <groupId>com.powsybl</groupId>
+                <artifactId>powsybl-time-series-api</artifactId>
+                <version>${powsybl-time-series.version}</version>
+            </dependency>
+            <!-- FIXME: to be removed at next gridsuite-dependencies upgrade  -->
+            <dependency>
+                <groupId>com.powsybl</groupId>
+                <artifactId>powsybl-time-series-dsl</artifactId>
+                <version>${powsybl-time-series.version}</version>
+            </dependency>
```

Comment:
Add :

       <dependency>
            <groupId>com.powsybl</groupId>
            <artifactId>powsybl-dynawo-dsl</artifactId>
            <version>${powsybl-dynawo.version}</version>
        </dependency>

        <dependency>
            <groupId>com.powsybl</groupId>
            <artifactId>powsybl-dynawo-simulation</artifactId>
            <version>${powsybl-dynawo.version}</version>
        </dependency>

Reply:
DONE in https://github.com/gridsuite/dynamic-simulation-server/pull/122/commits/feed5a67068f4a0365a22891c16cb4c7e0887f45